src/frontend/src/Components/Map.scss

```diff
@@ -99,7 +99,7 @@
         right: 1rem;
         padding: 0;
         min-width: 0;
-        line-height: 1.25rem;
+        line-height: 8.25rem;
```

Comment:
This looks a bit alarming to me as 'line-height' is usually used for the spacing between the lines of text in one block/element. I would look into using padding or margin instead.